### DISCUSSION TITLE
[ask] custom interface of web3modal

### DISCUSSION THREAD

**Discussion by ebennaga** [WalletConnect/walletconnect-monorepo#3470] [Developer Support] [answered]
i want to ask something, is it possible to custom interface ?
from here 
![modal web3](https://github.com/WalletConnect/walletconnect-monorepo/assets/2337413/0ef22663-3a5d-491e-888d-6250de20ede5)

to here
![to web3](https://github.com/WalletConnect/walletconnect-monorepo/assets/2337413/0621e678-2d03-4731-aea9-506155b504d7)

**Comment by glitch-txs** [comment]
Hi, yes, here are the different theming options
https://docs.walletconnect.com/2.0/web3modal/react/wagmi/theming

**Reply by ebennaga** [reply]
Thanks, how to hide qr code in web3modal?

**Reply by glitch-txs** [reply, marked as answer]
you can't do that on v2, but it's more organized on v3, it's still in Alpha but you can check [here](https://docs.walletconnect.com/2.0/web3modal/v3/about)
